wagtail/admin/templatetags/wagtailadmin_tags.py

```diff
@@ -699,19 +699,20 @@ def timesince_simple(d):
     1 week, 1 day ago -> 1 week ago
     0 minutes ago -> just now
     """
+    # Note: Duplicate code in timesince_last_update()
     time_period = timesince(d).split(",")[0]
     if time_period == avoid_wrapping(_("0 minutes")):
-        return _("Just now")
+        return _("just now")
     return _("%(time_period)s ago") % {"time_period": time_period}
 
 
 @register.simple_tag
 def timesince_last_update(
-    last_update, time_prefix="", user_display_name="", use_shorthand=True
+    last_update, show_time_prefix=False, user_display_name="", use_shorthand=True
```

Comment:
- naming ok?
- position ok?

Reply:
We could use `show_preposition` but that may be harder to understand haha.

I think this is fine - based on the legacy usage, I think the arg position is fine also.

Thanks for calling this out for a double check.